components/contrib/google-cloud/notebooks/component.yaml

```diff
@@ -0,0 +1,377 @@
+name: Execute notebook
+description: Function that creates the component file `output_component_file`.
```

Comment:
I would recommend adding full API description here since we use the description in yaml file to generate the API documentation. this is not a blocker though highly recommended as a follow up.

see https://google-cloud-pipeline-components.readthedocs.io/en/google-cloud-pipeline-components-0.1.6/

Reply:
For now, I changed the first line to be more descriptive.
Would you recommend to hardcode the description in the yaml file? `create_component_from_func` seems to keep only the first line of the function's comment. 

For example: 
- [aiplatform/batch_predict_job](https://github.com/kubeflow/pipelines/blob/master/components/google-cloud/google_cloud_pipeline_components/aiplatform/batch_predict_job/component.yaml#L3) has a full description (but no Python files around)
- [experimental/AutoML](https://github.com/kubeflow/pipelines/blob/master/components/google-cloud/google_cloud_pipeline_components/experimental/AutoML/Tables/Create_dataset/from_CSV/component.py#L20) runs into the same problem as I have (only one line is kept)